ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_rounding.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+from hypothesis import given, strategies as st
+
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy_tests.test_ivy.test_frontends.test_numpy.helpers as np_frontend_helpers
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_numeric_dtypes),
```

Comment:
We should now be using ivy.current_backend().valid_dtypes inside our helper functions, instead of ivy_np.valid_dtypes.